src/WattleScript.Interpreter/Extensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using WattleScript.Interpreter.Debugging;
+
+namespace WattleScript.Interpreter
+{
+    public static class Extensions
+    {
+        public static string GetSourceCode(this SourceRef[] refs, string fullSourceCode)
```

Comment:
`GetSourceCode` is a little vague here. Maybe `GetSourceFragment` or something that specifies it's taking a substring?

Reply:
Renamed in c72ec015165eb75af13745bde0f2a62be2b3d644